tests/src/system/basic/WskSdkTests.scala

```diff
@@ -94,7 +94,7 @@ class WskSdkTests
         val scriptfilename = "wsk_cli_bash_completion.sh"
         var scriptfile = new File(dir.getPath(), scriptfilename)
         try {
-            val stdout = wsk.cli(Seq("sdk", "install", "bashauto"), workingDir = dir, expectedExitCode = SUCCESS_EXIT).stdout
+            val stdout = wsk.cli(wskprops.overrides ++ Seq("sdk", "install", "bashauto"), workingDir = dir, expectedExitCode = SUCCESS_EXIT).stdout
```

Comment:
I made some updates to get rid of the API host requirement for the Bash autocomplete generation. Good catch.